.github/workflows/snapshot-test.yaml

```diff
@@ -11,6 +11,10 @@ on:
         required: false
         default: false
 
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  GITHUB_PR_NUM: ${{ github.event.pull_request.number }}
+
```

Comment:
Why are those environment variables needed even though the workflow has not changed?

Reply:
I just figured out that without this impact analysis does not work, so I implemented these variables to other workflows as well as part of this PR even though it is not directly related.